modules/infra/datapath/gr_slow_path.h

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: BSD-3-Clause
+// Copyright (c) 2024 Christophe Fontaine
+
+#ifndef _GR_INFRA_SLOW_PATH
+#define _GR_INFRA_SLOW_PATH
+
+#include "gr_mbuf.h"
+
+#include <rte_graph.h>
+
+GR_MBUF_PRIV_DATA_TYPE(slow_path_input_mbuf_data, { void *data; });
+
+void gr_slow_path_input_add_handler(uint16_t type, const char *node_name);
+int slow_path_post(uint16_t type, void *data);
```

Comment:
Maybe restrict slow path type to uint8_t ? Will we ever need more than 255 slow path handlers?